src/Compilers/CSharp/Test/Syntax/Parsing/LambdaParameterParsingTests.cs

```diff
@@ -5686,11 +5686,11 @@ public void ScopedAsParameterName_05()
             EOF();
         }
 
-        [Fact, WorkItem(63469, "https://github.com/dotnet/roslyn/issues/63469")]
-        public void ScopedAsParameterName_06()
+        [Theory, CombinatorialData, WorkItem("https://github.com/dotnet/roslyn/issues/63469")]
+        public void ScopedAsParameterName_06(bool isCSharp13)
         {
             string source = "(scoped scoped) => { }";
-            UsingExpression(source);
+            UsingExpression(source, isCSharp13 ? TestOptions.Regular13 : null);
```

Comment:
In C# 14, the `scoped` should be parsed as identifier. But it looks like passing `null` does not parse with Preview but with Latest. So consider testing this with LangVersion=Preview.

Reply:
I would still like to see this tested with LangVersion=Preview